js/blocks/loader/advanced.js

```diff
@@ -0,0 +1,29 @@
+/* global blockLab */
+
+const { InspectorAdvancedControls } = wp.editor;
+const { DotTip } = wp.nux;
+const { sprintf, __ } = wp.i18n;
+
+const inspectorControls = ( props, block ) => {
+	if ( '-1' === blockLab.authorBlocks.indexOf( block.name ) ) {
+		return;
+	}
+
+	const tip = sprintf(
+		__( 'The Additional CSS Class can be included in your block template with the %1$s field.', 'block-lab' ),
```

Comment:
No need to change this, but `%s` is probably alright instead of `%1$s`

Reply:
Good idea to have `'<code>className</code>'` in the second argument, so it can't change in translation.

Reply:
I always prefer specifying the placeholder order, due to this coding standard: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/e73a10e1c12ce07a12c3a8756efc54701458d23b/WordPress/Sniffs/WP/I18nSniff.php#L65

Reply:
Ah, interesting